src/shared/components/case-editor/services/event-completion-state-machine.service.spec.ts

```diff
@@ -0,0 +1,231 @@
+import { TestBed } from '@angular/core/testing';
+import { ActivatedRoute, Router } from '@angular/router';
+import { RouterTestingModule } from '@angular/router/testing';
+import { StateMachine } from '@edium/fsm';
+import { of } from 'rxjs';
+import { WorkAllocationService } from '.';
+import { AbstractAppConfig } from '../../../../app.config';
+import { Task } from '../../../domain/work-allocation/Task';
+import { HttpErrorService, HttpService } from '../../../services';
+import {
+  EventCompletionComponentEmitter,
+  EventCompletionStateMachineContext,
+  EventCompletionStates
+} from '../domain';
+import { EventCompletionStateMachineService } from './event-completion-state-machine.service';
+import createSpyObj = jasmine.createSpyObj;
+
+describe('EventCompletionStateMachineService', () => {
+  const API_URL = 'http://aggregated.ccd.reform';
+  let service: EventCompletionStateMachineService;
+  let stateMachine: StateMachine;
+  let mockSessionStorageService: any;
+  let appConfig: any;
```

Comment:
I know these are just tests but we do know the `Types`  names of these variables can we annotate them with `Actual Types`  instead of `any`

Reply:
I have managed to use the types here :)